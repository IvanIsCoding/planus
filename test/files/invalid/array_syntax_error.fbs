struct Foo {
  x: [bool; 4];
}