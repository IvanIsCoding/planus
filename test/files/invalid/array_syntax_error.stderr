[0m[1m[38;5;9merror[0m[1m: unrecognized token `;`, expected one of ":", "]"[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/array_syntax_error.fbs:2:11
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: [bool[0m[31m;[0m 4];
  [0m[34mâ”‚[0m           [0m[31m^[0m

Error: Bailing because of errors
